### PR TITLE
Remove ExecScopeProxy

### PR DESCRIPTION
**Pull request by Juan-M-V**
# Remove ExecScopeProxy

## Description

Removes the remaining proxy

## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

**Review by fmoletta — COMMENTED** [review]
Great Work!

File: src/types/exec_scope.rs [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
+};
+use num_bigint::BigInt;
+use std::{any::Any, cell::RefCell, collections::HashMap, rc::Rc};
 
 pub struct ExecutionScopes {
     pub data: Vec<HashMap<String, Box<dyn Any>>>,
 }
 
 impl ExecutionScopes {
+    // USED
```
[the remark this answers is not in the document]
Comment (Juan-M-V) [reply src/types/exec_scope.rs]: Resolved: https://github.com/lambdaclass/cairo-rs/pull/472/commits/08ff683ba9f4257d34baab53c65178948b635c28

File: src/types/exec_scope.rs [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
     pub fn new() -> ExecutionScopes {
         ExecutionScopes {
             data: vec![HashMap::new()],
         }
     }
 
+    // USED
```
[the remark this answers is not in the document]
Comment (Juan-M-V) [reply src/types/exec_scope.rs]: Resolved: https://github.com/lambdaclass/cairo-rs/pull/472/commits/08ff683ba9f4257d34baab53c65178948b635c28

File: src/types/exec_scope.rs [hunk 3]
```diff
@@ -1,21 +1,29 @@
...
     pub fn enter_scope(&mut self, new_scope_locals: HashMap<String, Box<dyn Any>>) {
         self.data.push(new_scope_locals);
     }
 
+    // USED
```
[the remark this answers is not in the document]
Comment (Juan-M-V) [reply src/types/exec_scope.rs]: Resolved: https://github.com/lambdaclass/cairo-rs/pull/472/commits/08ff683ba9f4257d34baab53c65178948b635c28

**Review by fmoletta — APPROVED** [review]

**Review by entropidelic — APPROVED** [review]

**Pull request merged**